ops/model.py

```diff
@@ -346,6 +353,17 @@ def set_workload_version(self, version: str) -> None:
                 type(version).__name__, version))
         self._backend.application_version_set(version)
 
+    @property
+    def containers(self) -> typing.Dict:
+        """Return a dict of (Kubernetes) containers indexed by name."""
```

Comment:
s/Kubernetes//

Specifically this logic should be agnostic. This is a list of containers associated with the unit, k8s or not.